### ISSUE TITLE
Right click a Word Model file (*.dotx) and choosing Open create a new file instead of opening the template

### ISSUE THREAD

**Issue by JeanPhilippeGeorge** [files-community/Files#4279]
**Explain the bug**
Right click a Word Model file (*.dotx) and choosing Open create a new file instead of opening the template. So when you save the document you create a new document instead of saving the template.

**To reproduce**
1. Open a folder with at least a .dotx file
2. Right-click on the .dotx file
3. Choose Open
4. See that the Word's title bar shows : documentX - Word

**Expected behavior**
Open the template, so I can modify it and save it. Not creating a new document.
Doing the same in the file Explorer does the correct / expected behavior (look at the title bar for comparison).

**Logfile**
.

**Screenshots**
Right-click (note : Ouvir is Open in french):
![image](https://user-images.githubusercontent.com/36743032/112347877-8a345c80-8cc7-11eb-82ec-aa464cdf9742.png)

Show this:
![image](https://user-images.githubusercontent.com/36743032/112347276-fc587180-8cc6-11eb-9502-f1ad4866274a.png)

Instead of 
![image](https://user-images.githubusercontent.com/36743032/112347290-00848f00-8cc7-11eb-923e-7e7979d51dba.png)

**System Information**
 - Windows 10 20H2 19042.804
 - App version: 1.5.0.0

**Additional comment**
.

**Comment by yaichenbaum** [comment]
Is this still an issue in the latest preview build? https://files.community/download/preview

**Comment by JeanPhilippeGeorge** [comment]
Yes.
And I tried .dotm too (model wih macro): same problem.

**Comment by yaichenbaum** [comment]
Can you confirm the version and os number?

**Comment by JeanPhilippeGeorge** [comment]
FIles: 2.3.40.0
Windows 10: 21H2 (OS Build 19044.1889)

I also checked that it works with the Windows Files Explorer, and it does. 

In Windows file eplorer the right click menu shows New, Open and Open with.

Files shows only Open and Open with

**Comment by yaichenbaum** [comment]
I'm having the same issue in File Explorer. Can you upload a screen recording of File Explorer compared to Files on your device?

**Comment by JeanPhilippeGeorge** [comment]
Not sure of what your want. 

The main difference is that when clicking on Open in Files it does exactly the same than when clicking New. In Windows Files Explorer it opens it (see picture on top for the difference).

I'm guessing that there is a problem (cache?) when you are reading the values in the registry.

If not mistaken, those values come from here (get the default value):
![image](https://user-images.githubusercontent.com/36743032/196167045-47343a66-5c15-4974-988a-148770b24cb0.png)

Then follow to Word.Template.12, then look in the shell folder, then for the Open command (in our case), then the command subfolder which gives the command line magic to open the file in Word.
![image](https://user-images.githubusercontent.com/36743032/196167146-0cb5de69-d320-4e49-b498-ef372ae73416.png)


For new, arguments are different:
![image](https://user-images.githubusercontent.com/36743032/196169147-2f9be657-7b6b-4f79-8dd1-f3c07585cdfb.png)

All those values are dependent of the file type (.dotx).

As there is the same problem with .dotm. Maybe you read the value for .docx and then use them for all Word files (dotx, dotm, ...)? Or some weird cache problem?

**Comment by yaichenbaum** [comment]
> Not sure of what your want.

I'd like to see screen recording of File Explorer compared to Files on your device so that I can verify I'm using the same steps on my end to reproduce the issue.

**Comment by JeanPhilippeGeorge** [comment]
Files : https://1drv.ms/v/s!AveH8oTUFVRYk_BBrNddr8qP11zF9g?e=MMlzu1

Windows Explorer : https://1drv.ms/v/s!AveH8oTUFVRYk_BC7jLJBTPWkgNDNw?e=4kdcZ6

**Comment by yaichenbaum** [comment]
When I try opening the dotx file from File Explorer, it creates a copy rather than open the existing template.

**Comment by JeanPhilippeGeorge** [comment]
Yes, it does what New does instead of opening the file. New is the default operation in Windows Explorer
